.devcontainer/Dockerfile

```diff
@@ -1,6 +1,8 @@
 ARG VARIANT=17-bullseye
 FROM mcr.microsoft.com/vscode/devcontainers/java:0-${VARIANT}
 
+RUN sudo apt update && sudo apt install -y maven && sudo apt-get clean
```

Comment:
Is it required?

Reply:
Yes, it was needed for my development work

Reply:
It is not required if you follow the instructions in the readme - the Maven wrapper is installed in the source code.